test/unit/data/custom-metadata-json/ambiguous_scenarios_specified_v1.json

```diff
@@ -0,0 +1,100 @@
+        "keyword": "Scenario",
+        "line": 5,
+        "name": "Ambiguous Johnny wants to see ambiguous scenarios",
+        "steps": [
+          {
+            "arguments": [],
+            "keyword": "Before ",
+            "result": {
+              "status": "passed",
+              "duration": 1352242
+            },
+            "hidden": true,
+            "match": {
+              "location": "/Users/wswebcreation/deTesters/protractor-cucumber-typescript-boilerplate/node_modules/cucumber-tsflow/src/BindingDecorator.ts:83"
+            }
+          },
+          {
+            "arguments": [],
+            "keyword": "Given ",
+            "name": "Ambiguous Johnny visits the Angular homepage",
+            "result": {
+              "status": "passed",
+              "duration": 1930520
+            },
+            "line": 6,
+            "match": {
+              "location": "/Users/wswebcreation/deTesters/protractor-cucumber-typescript-boilerplate/node_modules/cucumber-tsflow/src/BindingDecorator.ts:157"
+            }
+          },
+          {
+            "arguments": [],
+            "keyword": "When ",
+            "name": "he clicks on the second button",
+            "result": {
+              "status": "ambiguous"
+            },
+            "line": 7
+          },
+          {
+            "arguments": [],
+            "keyword": "Then ",
+            "name": "he expects that something happens",
+            "result": {
+              "status": "skipped"
+            },
+            "line": 8,
+            "match": {
+              "location": "/Users/wswebcreation/deTesters/protractor-cucumber-typescript-boilerplate/node_modules/cucumber-tsflow/src/BindingDecorator.ts:163"
+            }
+          },
+          {
+            "arguments": [],
+            "keyword": "After ",
+            "result": {
+              "status": "passed",
+              "duration": 349939
+            },
+            "hidden": true,
+            "match": {
+              "location": "/Users/wswebcreation/deTesters/protractor-cucumber-typescript-boilerplate/node_modules/cucumber-tsflow/src/BindingDecorator.ts:201"
+            }
+          },
+          {
+            "arguments": [],
+            "keyword": "After ",
+            "result": {
+              "status": "passed",
+              "duration": 263324
+            },
+            "hidden": true,
+            "match": {
+              "location": "/Users/wswebcreation/deTesters/protractor-cucumber-typescript-boilerplate/node_modules/cucumber-tsflow/src/BindingDecorator.ts:87"
+            }
+          }
+        ],
+        "tags": [],
+        "type": "scenario"
+      }
+    ],
+    "id": "ambiguous-scenarios-specified-v1",
+    "keyword": "Feature",
+    "line": 1,
+    "name": "Ambiguous scenarios specified V1",
+    "tags": [],
+    "uri": "/Users/wswebcreation/deTesters/protractor-cucumber-typescript-boilerplate/e2e-tests/features/ambiguous.feature",
+    "metadata": [
```

Comment:
I think this can be removed

Reply:
Do you mean that the file can be removed? I just overwrote the `metadata` field to re use it.

Reply:
No this is good 👍